### ISSUE TITLE
Upgrade Pokemon Error Not Found

### ISSUE THREAD

**Issue by ghost** [FeroxRev/Pokemon-Go-Rocket-API#46]
when doing 
_client.Inventory.UpgradePokemon(PokemonId);
it always returns that pokemon was not found i am supplying the correct pokemon id so maybe a bug in the api?

**Comment by MonsterMMORPG** [comment]
same error here

just reproduced

**Comment by FeroxRev** [comment]
Did you supply the ulong id and not the pokemonid enum?

**Comment by MonsterMMORPG** [comment]
@FeroxRev 

yep i have provided Id which is ulong

i have exactly tested as evolve which works

```
        var evolveResponse2 = await session.Client.Inventory.UpgradePokemon(pokemon.Id);
                var evolveResponse = await session.Client.Inventory.EvolvePokemon(pokemon.Id);
```

**Comment by ghost** [comment]
yes i did supply the ulong id and it doesnt return properly

**Comment by neoz** [comment]
You may make wrong param, Id not PokemonId.
I have try the UpgradePokemon with successful.

**Comment by MonsterMMORPG** [comment]
@neoz i have tried with correct parameter

using necro bot

see this

  var evolveResponse2 = await session.Client.Inventory.UpgradePokemon(pokemon.Id);
                var evolveResponse = await session.Client.Inventory.EvolvePokemon(pokemon.Id);

**Comment by kanfkasndklasdn** [comment]
I've done the same as MonsterMMORPG. Pokemon evolve succesfully, upgrade was wrong with pokemon not found error...

**Comment by neoz** [comment]
@MonsterMMORPG ,
I'm not so sure, the only different is I'm using the lib POGOProtos build from github (link: https://github.com/AeonLucid/POGOProtos). Not using the lib from NuGet. The proto in NuGet lib may not the lastest one.

**Comment by MonsterMMORPG** [comment]
@neoz yes that might be the reason

**Comment by neoz** [comment]
@MonsterMMORPG or maybe you evolve the pokemon and then powerup without reupdate new id after evolve ?

**Comment by MonsterMMORPG** [comment]
@neoz  it is not related to evolve. even removing evolve line doesnt do a thing . i just wanted to show i am properly providing id

**Comment by ghost** [comment]
Im going to have another look at this to see if we were missing anything i will update if i find anything

**Comment by ghost** [comment]
Still not working i am getting a ulong id such as 
15233370587308033284
which i got from getting my pokemon list which had its id name etc
then put that in the upgrade pokemon and got pokemon not found

**Comment by kanfkasndklasdn** [comment]
I still can't do it too

**Comment by MonsterMMORPG** [comment]
i think it is now obvious that our protos are problematic since neoz is able to do with other protos

**Comment by ghost** [comment]
@neoz  mind sharing your dll file please?

**Comment by kanfkasndklasdn** [comment]
I've build it, but I don't know how to generate the .dll from .cs's

**Comment by vmcall** [comment]
Could anyone please share the updated proto shit? :+1: That'd be great

**Comment by FeroxRev** [comment]
Can you try again with latest commits?

**Comment by ghost** [comment]
Just tried again and same error pokemon not found

**Comment by kanfkasndklasdn** [comment]
Now works for me!

**Comment by ghost** [comment]
Let me try again aha maybe i missed something

**Comment by ghost** [comment]
Ah it works perfectly now thank you 
problem was i still had the old 1.3 in my other project so caused errors
